controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -388,7 +368,12 @@ def test_create_snapshot_create_volume_error(self, mock_warning):
             CLIFailureError("Failed")]
 
         with self.assertRaises(CLIFailureError):
-            self.svc.create_snapshot("test_snapshot", "source_volume")
+            self.svc.create_snapshot("source_volume_id", "test_snapshot", "pool1")
+
+    def test_create_snapshot_source_not_found_error(self):
```

Comment:
what about illegal object id?

Reply:
(same for all mediators)

Reply:
(same for server tests)

Reply:
done

Reply:
I still couldn't find coverage for it in xiv and server - could you point me to them?

Reply:
there is now

Reply:
same for "get_snapshot"

Reply:
done